rpcd/playbooks/verify-maas.yml

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 
 - name: Verify MaaS setup
-  hosts: hosts
+  hosts: shared-infra_hosts:compute_hosts:storage_hosts:swift_hosts:log_hosts:all_containers
```

Comment:
``verfiy-maas.yml`` doesn't include the all_containers group like ``setup-maas.yml`` does. It can be omitted here.